Models/MCTS_Util.py

```diff
@@ -6,27 +6,52 @@
 
 def create_default_mapping():
     mappings = []
+    mask = []
 
-    for a in range(38):
-        for b in range(37):
+    for b in range(37):
```

Comment:
For the future, avoiding these kind of magic numbers is a must. Is this board size?

Reply:
It's the board size plus the bench size. The 38 adds one for the sell action. It's a repeat of what is in the action class basically.

Reply:
So you see how having 2 constants like (board size and bench size) could make things way more readeable